src/components/popup/home/Balance.tsx

```diff
@@ -25,7 +25,10 @@ import Arweave from "arweave";
 import { removeDecryptionKey } from "~wallets/auth";
 import { findGateway } from "~gateways/wayfinder";
 import BigNumber from "bignumber.js";
-import { retryWithDelay, retryWithDelayAndTimeout } from "~utils/retry";
+import {
+  retryWithDelay,
+  retryWithDelayAndTimeout
+} from "~utils/promises/retry";
```

Comment:
Changed all these imports as I've started moving some files to their own directory to better organize files once I start adding tests.

Reply:
👍